lexicons/com/atproto/identity/refreshIdentity.json

```diff
@@ -0,0 +1,30 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.refreshIdentity",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Request that the server re-resolve an identity (DID and handle). The server may ignore this request, or require authentication, depending on the role, implementation, and policy of the server.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["identifier"],
```

Comment:
`"required": ["hostname"]` ?

Reply:
oh, good catch! though the other way around (field name should be "identifier")